style.css

```diff
@@ -369,4 +369,78 @@ body {
     justify-content: space-between;
     column-gap: 1.6rem;
   }
+
+  .contact-me {
+    background-image: url("./assets/images/contact-form-shapes-mobile.png");
+    background-color: #6070ff;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 10.4rem  2.4rem 0.8rem 2.4rem;
+    margin-top: 17.685rem;
+    border-top-left-radius: 6rem;
+  }
+
+  .contact-me h2 {
+    flex-basis: 100%;
+    font-weight: 700;
+    font-size: 4rem;
+    line-height: 5.2rem;
+    color: #fff;
+    margin-bottom: 1.2rem;
+    text-align: center;
+  }
+
+  .contact-me p {
+    color: #ebebff;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.8rem;
+    flex-basis: 100%;
+    text-align: center;
+    margin-bottom: 5.2rem;
+  }
+
+  .contact-me button {
+    width: 14.2rem;
+    height: 4.8rem;
+    border-radius: 0.8rem;
+    color: #6070ff;
+    background-color: #fff;
+    border: 0;
+    font-size: 1.7rem;
+    font-weight: 500;
+    line-height: 2.4rem;
+    margin-bottom: 3.2rem;
+  }
+
+  .contact-me form {
+    flex-basis: 100%;
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+    width: 32.7rem;
+  }
+
+  .contact-me form input {
+    padding: 1.5rem 3.8rem 1.5rem 1.6rem;
+    border-radius: 0.8rem;
+    border: 0;
+  }
```

Comment:
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching because the height should be **48px** which looks perfect on the mobile screen instead of **18px**. Please have a look at the below comparison table to understand the problem in a better way through visualization.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 22-30-06](https://github.com/AjroudiRida/Portfolio/assets/66476329/dadcb37c-5993-48c4-b592-796334261a21)

</td>

<td>

![Screenshot from 2023-06-06 21-02-32](https://github.com/JoyTaribagshaw/My-Portfolio/assets/66476329/65578e16-f1b9-47eb-b514-1c53b1c7ff20)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)